src/main/scala/uk/gov/nationalarchives/dp/client/Entities.scala

```diff
@@ -1,27 +1,37 @@
 package uk.gov.nationalarchives.dp.client
 
 import cats.MonadError
+import cats.implicits.catsSyntaxOptionId
 
 import java.util.UUID
 
 object Entities {
-  case class Entity(entityType: String, ref: UUID, title: Option[String], deleted: Boolean, path: String)
+  case class Entity(
+      entityType: Option[String],
+      ref: UUID,
+      title: Option[String],
+      deleted: Boolean,
+      path: Option[String]
+  )
 
   def fromType[F[_]](entityType: String, ref: UUID, title: Option[String], deleted: Boolean)(implicit
       me: MonadError[F, Throwable]
   ): F[Entity] = entityType match {
     case "IO" =>
       me.pure {
-        Entity("IO", ref, title, deleted, "information-objects")
+        Entity("IO".some, ref, title, deleted, "information-objects".some)
```

Comment:
Couldn't you just do `Some("IO")`? or does `.some` offer another advantage? I was thinking that it would be one fewer import/Cats dependency and probably saves a tiny bit of time having to do the implicit conversion each time (albeit probably negligible).

Reply:
It's just syntax sugar, I wouldn't bother usually but we've already got a load of cats implicits anyway and the network calls are so slow, the time doesn't matter. 
I prefer it because it's fewer brackets.